translations/monero-core_eo.ts

```diff
@@ -29,14 +29,19 @@
         <translation>paga-ID &lt;font size=&apos;2&apos;&gt;(Optional)&lt;/font&gt;</translation>
     </message>
     <message>
+        <location filename="../pages/AddressBook.qml" line="105"/>
```

Comment:
Can you help to explain where this translation came from?
Does not match what I see on line 105 of `AddressBook.qml`:
https://github.com/monero-project/monero-core/blob/master/pages/AddressBook.qml#L105

I see [Hebrew also has the same section](/monero-project/monero-core/blob/master/translations/monero-core_he.ts#L38), but that is the only other language.  I'm guessing this slipped in by accident?

If so, you can remove entire `<message>`...`</message>`.
The English here does not make much sense anyway.

Reply:
Hi, I did improve on @dnaleor 's translation, he should be able to tell you where he found it. I'm not confortable removing this part myself, am I the only one who can do it ?

Reply:
Yeah, I am not a project owner, so I cannot remove for you unfortunatly.